components/sources/cu_hesai/src/lib.rs

```diff
@@ -235,7 +165,8 @@ mod tests {
 
         xt32.reftime = (datetime, clock.now());
 
-        while streamer.send_next() {
+        // 1076 is the expected payload size for Hesai XT32
+        while streamer.send_next::<1076>() {
```

Comment:
Not a fan of magic static value, since its statically deductible from packet struct  with a size_of::<lidar_parser::Packet>